### DISCUSSION TITLE
How to set shortcut for - Move selected text from page

### DISCUSSION THREAD

**Discussion by supertoothy** [zim-desktop-wiki/zim-desktop-wiki#1792] [Q&A] [answered]
**_Moving selected Text_** from the page is a very useful feature. I often find myself composing long form text on a single page, then breaking them down into separate pieces into their own pages.

Currently this function can be accessed via the mouse right click context menu. I want to set up a keyboard shortcut for this, but cannot find this in keybindings. What am I missing?

thanks!

**Comment by Nick222** [comment]
It seems to me that there will be many such requests.
It is much better to have Zim support some scripting language.

**Comment by jaap-karssenberg** [comment]
Indeed, should be regular action in order to allow accelerator key binding

@nick python is a scripting language, and we support plugins, so no need
for another layer on top ....

Op vr 10 dec. 2021 10:23 schreef Nick222 ***@***.***>:

**Comment by Nick222** [comment]
I think it is not so simple and quick to write Python-script to small user-operation...

**Comment by jaap-karssenberg** [comment, marked as answer]
Sorry Nick, but I'm not going to entertain the idea of yet another
customization mechanism.

@gsupreeth please open a ticket requesting this action to be a regular
action so a keybinding can be defined

**Reply by introt** [reply]
Marking this as the answer as there's now an open issue tracking this

**Comment by supertoothy** [comment]
Thanks. Done!
